frontend/src/utils/Validators.ts

```diff
@@ -489,8 +480,21 @@ export function findFaultInMittelschuleForSave(mittelschule: MittelschuleDto): s
   return findFaultInInfrastruktureinrichtung(mittelschule);
 }
 
-export function findFaultInVerteilungWohneinheitenAbfragevariante(
-  abfragevariante: AnyAbfragevarianteDto,
+/**
+ * Überprüft, ob die Verteilung der Wohneinheiten und Geschossfläche Wohnen auf Baugebiete bzw. Bauraten valide ist.
+ * WE bzw. GF gelten als "verteilt", wenn bei mindestens einem Baugebiet bzw. Baurate eine Angabe zu ihnen gemacht wurde.
+ * WE bzw. GF gelten als "korrekt verteilt", wenn zusätzlich ihre Summe in den Baugebieten bzw. Bauraten der Gesamtanzahl entspricht.
+ * Die Verteilung ist valide, wenn WE und GF korrekt verteilt wurden.
+ * Ebenso ist die Verteilung valide, wenn entweder WE oder GF korrekt verteilt wurde, aber der andere Wert nicht verteilt wurde.
+ *
+ * @param abfragevariante zur Validierung.
+ * @return ob die Verteilung entsprechend der Beschreibung valide ist.
+ */
+export function findFaultInWeGfVerteilungAbfragevariante(
```

Comment:
Auch in den Klassen zur Frontendvalidierung in den Methodendocs oder in der Methode an den entsprechenden Stellen kurz den Grund für die Unterscheidung zwischen technischen und nichttechnischen Baugebieten beschreiben.

Siehe auch Backend: https://github.com/it-at-m/isi-backend/pull/158#discussion_r1556997535